Projections/Win2D/Win2D.csproj

```diff
@@ -0,0 +1,84 @@
+    <ProjectReference Include="..\..\WinRT.Runtime\WinRT.Runtime.csproj" />
+    <ProjectReference Include="..\..\cswinrt\cswinrt.vcxproj" />
+    <ProjectReference Include="..\Windows\Windows.csproj" />
+    <PackageReference Include="Win2D.uwp" Version="[$(Win2DVersion)]" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Generated Files\" />
+  </ItemGroup>
+  
+  <Target Name="GenerateProjection" AfterTargets="ResolveAssemblyReferences" BeforeTargets="DispatchToInnerBuilds;Build" Condition="'$(GenerateTestProjection)' == 'true'">
+    <ItemGroup>
+      <ReferenceWinMDs Include="@(ReferencePath)" Condition="'%(ReferencePath.WinMDFile)' == 'true'" />
+    </ItemGroup>
+    <PropertyGroup>
+      <CsWinRTVerbosity>high</CsWinRTVerbosity>
+      <CsWinRTResponseFile>$(IntermediateOutputPath)cswinrt_test.rsp</CsWinRTResponseFile>
+      <CsWinRTCommand>$(CsWinrtExe) %40"$(CsWinRTResponseFile)"</CsWinRTCommand>
+    </PropertyGroup>
+    <PropertyGroup>
+      <CsWinRTParams>
+-verbose
+-in 10.0.18362.0
+-in @(ReferenceWinMDs->'"%(FullPath)"', ' ')
+-out "$(ProjectDir)Generated Files"
+-include Microsoft
+-exclude Windows
+# Include types excluded per Microsoft.Windows.SDK.WinUI.Contracts NuGet
+#-include Windows.UI.Colors
+#-include Windows.UI.IColors
+#-include Windows.UI.ColorHelper
+#-include Windows.UI.IColorHelper
+-include Windows.UI.Xaml
+      </CsWinRTParams>
+    </PropertyGroup>
+    <MakeDir Directories="$(ProjectDir)Generated Files" />
+    <WriteLinesToFile File="$(CsWinRTResponseFile)" Lines="$(CsWinRTParams)" Overwrite="true" WriteOnlyWhenDifferent="true" />
+    <Message Text="$(CsWinRTCommand)" Importance="$(CsWinRTVerbosity)" />
+    <Exec Command="$(CsWinRTCommand)" />
+  </Target>
+
+  <Target Name="IncludeProjection" BeforeTargets="CoreCompile" AfterTargets="GenerateProjection">
+    <ItemGroup>
+      <Compile Include="$(ProjectDir)Generated Files/*.cs" Exclude="@(Compile)" />
+      <!--Remove references to projection source winmds to prevent compile conflict warnings-->
+      <ReferencePathWithRefAssemblies Remove="@(ReferencePathWithRefAssemblies)" Condition="%(ReferencePathWithRefAssemblies.Filename) == 'Microsoft.Graphics.Canvas'" />
+    </ItemGroup>
+  </Target>
+
+  <!--<Target Name="CreatePackage" AfterTargets="Build" Condition="'$(IsInnerBuild)' != 'true'" Inputs="$(OutDir)\netstandard2.0\Win2D.dll;$(OutDir)\netcoreapp5.0\Win2D.dll" Outputs="$(NugetBuildDir)Win2D.$(NugetTestVersion).nupkg">
+    <PropertyGroup>
+      <NugetCommandLine>$(SolutionDir).nuget\nuget pack Win2D.nuspec -Version $(NugetTestVersion) -OutputDirectory $(NugetBuildDir) -Properties Configuration=$(Configuration);Platform=$(Platform);Win2DVersion=$(Win2DVersion) -NonInteractive -Verbosity Detailed -NoPackageAnalysis</NugetCommandLine>
+    </PropertyGroup>
+    <Exec Command="$(NugetCommandLine)" />
+    <Exec Command="rd /s/q &quot;$(NugetCacheDir)Win2D\$(NugetTestVersion)&quot;" />

```

Comment:
I am assuming this is commented out intentionally?

Reply:
this folder is cruft - deleted